djangosaml2/middleware.py

```diff
@@ -69,6 +69,6 @@ def process_response(self, request, response):
                         path=settings.SESSION_COOKIE_PATH,
                         secure=settings.SESSION_COOKIE_SECURE or None,
                         httponly=settings.SESSION_COOKIE_HTTPONLY or None,
-                        samesite=None
+                        # samesite=None
```

Comment:
comment?

Reply:
removed the comment in the updated code.